src/test/resources/features/pre_and_post_installation_hooks.feature

```diff
@@ -11,10 +11,10 @@ Feature: Hooks
 		And the system is bootstrapped
 		And the candidate "grails" version "2.1.0" is available for download on "Linux" with architecture "x86_64"
 		And a "pre" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns successfully
-		And a "post" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns successfully
+#		And a "post" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns successfully
```

Comment:
Is it worth stripping this out of the step defs if we're not longer going to us it?